.github/workflows/convert-to-pdf.yml

```
@@ -0,0 +1,31 @@
+name: Wersje tekstowe to PDF
+# This workflow is triggered on pushes to the repository.
+on:
+  push:
+    branches:
+      - main
+    # Paths can be used to only trigger actions when you have edited certain files, such as a file within the /docs directory
+    paths:
+      - 'wersje tekstowe/**.md'
+      - 'wersje tekstowe/images/**'
+
+jobs:
+  converttopdf:
+    name: Build PDF
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

There's a newer version of checkout action 
- https://stackoverflow.com/questions/73473617/what-is-the-difference-between-github-actions-checkoutv3-and-v2
- https://github.com/actions/checkout

```suggestion
      - uses: actions/checkout@v3
```